boefjes/pyproject.toml

```diff
@@ -63,6 +63,7 @@ pydantic-settings = "^2.0.3"
 opentelemetry-instrumentation = "^0.41b0"
 # required by kat_webpage_analysis/check_images normalizer
 pillow = "^10.2.0"
+certifi = "^2024.2.2"
```

Comment:
If you use `poetry add` to update an indirect dependency it adds it to pyproject and you need to remove it by hand, as far as I know there isn't a command that updates a package that doesn't do this?

Reply:
You're right. Mistakenly assumed this was a top level dependency, but it isn't. I'll remove them from as a top-level dependency and make sure it's updated.